packages/uicore/src/utils/errorUtils.ts

```diff
@@ -7,15 +7,18 @@
 
 import { isEmpty } from 'lodash-es'
 
+const IGNORED_ERRORS = [
+  'Failed to fetch: The user aborted a request.',
+  'Failed to fetch: Failed to fetch',
+  'Failed to fetch: 504 Gateway Timeout',
+  "Failed to fetch: Failed to execute 'fetch' on 'Window': The user aborted a request.",
+  'Failed to fetch: The operation was aborted.'
+]
+
 export function shouldShowError(e: any): boolean {
   const hideMessagesForStatusCodes = [502, 503]
-  if (
-    e?.message === 'Failed to fetch: The user aborted a request.' ||
-    e?.message === 'Failed to fetch: Failed to fetch' ||
-    e?.message === 'Failed to fetch: 504 Gateway Timeout' ||
-    e?.message === "Failed to fetch: Failed to execute 'fetch' on 'Window': The user aborted a request." ||
-    hideMessagesForStatusCodes.includes(e?.status)
-  ) {
+  const message = (e?.message || '').trim()
+  if (IGNORED_ERRORS.includes(message) || hideMessagesForStatusCodes.includes(e?.status)) {
```

Comment:
Super-small optimisation, but you can just return this instead of having an `if`

```suggestion
  return !IGNORED_ERRORS.includes(message) && !hideMessagesForStatusCodes.includes(e?.status)
```